packages/react-components/react-skeleton/src/components/Skeleton/useSkeleton.ts

```diff
@@ -12,17 +12,22 @@ import type { SkeletonProps, SkeletonState } from './Skeleton.types';
  * @param ref - reference to root HTMLElement of Skeleton
  */
 export const useSkeleton_unstable = (props: SkeletonProps, ref: React.Ref<HTMLElement>): SkeletonState => {
+  //Props
+  const { animation = 'wave' } = props;
+
+  const root = getNativeElementProps('div', {
+    ref,
+    role: 'progressbar',
+    'aria-busy': 'true',
```

Comment:
```suggestion
    'aria-busy': true,
```

nit, This should also work